keps/sig-node/4622-topologymanager-max-allowable-numa-nodes/README.md

```diff
@@ -334,24 +331,25 @@ well as the [existing list] of feature gates.
 [existing list]: https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
 -->
 
-- [x] Feature gate (also fill in values in `kep.yaml`)
-  - Feature gate name: 
+- [X] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name:
     - `TopologyManagerPolicyBetaOptions`
-    - `TopologyManagerPolicyOptions`
+    - `TopologyManagerPolicyOptions` - going to be locked to true once GA
```

Comment:
no need to mention the locking, it's the standard process for each feature

Reply:
removed.